### DISCUSSION TITLE
Letters arranged vertically

### DISCUSSION THREAD

**Discussion by leminhhuy132** [PaddlePaddle/PaddleOCR#9840] [Q&A]
Does this model support letters arranged vertically as shown below? 
I have a data with letters arranged vertically, and i want to reg it
![image](https://user-images.githubusercontent.com/77653557/234871903-26c074ca-d3ef-42fb-822d-b6077af11073.png)

**Comment by nestorprojects** [comment]
I tried with model for recognition and bounding box, it detects as separate letters, even with some inbetween words like alamy,
Maybe other version of it are available that could do it with some tweaks, but thought might as well provide some info.. peace
